_includes/payment-state.md

```diff
@@ -0,0 +1,31 @@
+{% assign api_resource = include.api_resource | default: "`paymentorder`" %}
```

Comment:
Better to keep the variable plain and add the formatting within the text. I also think it should say `paymentorders` in plural for consistency.

```suggestion
{% assign api_resource = include.api_resource | default: "paymentorders" %}
```